src/pages/Home.js

```diff
@@ -16,11 +16,51 @@ export default function Home(props) {
     history.push('/list');
   }
 
+  function handleInputValue(e) {
+    setInputValue(e.target.value.trim());
```

Comment:
I like that you thought about handling the case of too many white spaces. What happens when a user enters `OrDer nor Sally` or `ORDER    nor             sally`? Github parsed out my spaces between the words. The second example is `order` then 3 spaces, `nor` then 2 spaces, then `sally`.

Reply:
I just tried inputting a token, and I love the idea of normalizing input! But I can see how it might be confusing or frustrating to a user why their input isn't going in the same way they're typing it (no spaces allowed). Maybe we can normalize in a different place? 

@yenly is one option to have the token normalized with no spaces/no capitalization/no punctuation, much like we do with the check for duplicates in `AddItem.js`? 

The user still inputs three separate words, but Firestore sees something like: `ordernorsally` and we normalize the input for `Join an existing list` in `Home.js`? 

If that's an acceptable solution, we can just copy/paste the Regex we use to check for duplicates.

Reply:
As I'm looking at this again, it's a good idea to trim the white spaces at the beginning and end of the `e.target.value` before saving into the state. The part to handle parsing `inputValue` before comparing to what's in Firestore should be done in `checkExistingToken` function. 

So if a user inputs `OrDer nor Sally` or `ORDER NOR  SALLY`, would they still be able to access the list? @jamesncox , your suggestion to normalize it to `ordernorsally` before saving to Firestore is a good idea. Each time before doing a get, you can normalize and know that you are comparing apples to apples.

Reply:
Thank you for addressing different scenarios! You both are correct, trimming at this point only works for one-word tokens. I noticed that @jamesncox extracted the regex into a function in his PR. Maybe it'd be a good idea to add it to a `lib` file, or lift to state, so we could also use it here?